bin/create-scxa-gene2experiment-schema.sh

```diff
@@ -46,15 +46,19 @@ curl -X POST -H 'Content-type:application/json' --data-binary '{
 
 #############################################################################################
 
-printf "\n\nDelete update processor"
+printf "\n\nDelete update processor "
 curl -X POST -H 'Content-type:application/json' --data-binary '{
-  "delete-updateprocessor":
-  {
-    "name": "'$CORE'_dedup"
+  "delete-updateprocessor": "'$CORE'_dedup"
+}' http://$HOST/solr/$CORE/config
+
+printf "\n\nDisable autoCreateFields (aka “Data driven schema”)"
```

Comment:
I think that we had this somewhere... https://github.com/ebi-gene-expression-group/index-scxa/blob/develop/bin/scxa-index-set-no-autocreate.sh but good to have it here. That script wasn't mentioned in the docs I guess, and probably there is no gain in having it separated.